lib/src/directory/link.dart

```diff
@@ -8,19 +8,28 @@ import 'package:s3i_flutter/src/utils/json_keys.dart';
 /// Represents a `Link` in the S3I-Directory data model.
 class Link implements JsonSerializableObject {
   /// Creates a new [Link] with the [association] and an optional [target].
-  Link(this.association, {this.target});
+  Link(this.association, {this.target, this.classString});
```

Comment:
I can't see an attribute `class` or `classString` of an `Link` in the latest data model of the S3I-Directory. We could add a `targetThing` which is currently missing, but in my opinion, this is never used so this is nothing with high prior.

<img width="445" alt="image" src="https://user-images.githubusercontent.com/47630370/175023708-214829d7-8e15-43f3-be97-c90eab6beaa1.png">